Source/Core/DolphinQt/FIFO/FIFOPlayerWindow.cpp

```diff
@@ -49,34 +52,47 @@ FIFOPlayerWindow::FIFOPlayerWindow(FifoPlayer& fifo_player, FifoRecorder& fifo_r
 
   UpdateControls();
 
-  m_fifo_player.SetFileLoadedCallback(
-      [this] { QueueOnObject(this, &FIFOPlayerWindow::OnFIFOLoaded); });
-  m_fifo_player.SetFrameWrittenCallback([this] {
-    QueueOnObject(this, [this] {
-      UpdateInfo();
-      UpdateControls();
-    });
-  });
+  m_fifo_player.SetFileLoadedCallback([this]
+                                      { QueueOnObject(this, &FIFOPlayerWindow::OnFIFOLoaded); });
```

Comment:
I specifically do not want this.. :(

Reply:
`AlignAfterOpenBracket: DontAlign` does something good, but it also seems to affect every single braced list.
`BinPackLongBracedList: true` would probably solve that, but it requires clang-format-21. :(